.changes/next-release/enhancement-ssm Session Manager-42633.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This is already been merged into v2 I believe: https://github.com/aws/aws-cli/blob/66af26fd0de98269959b5d570ebfd91e8018faef/CHANGELOG.rst#L40. We should be able to just delete this entry.